serverless_llm/serve/app_lib.py

```diff
@@ -50,7 +50,10 @@ async def register_handler(request: Request):
             raise HTTPException(
                 status_code=500, detail="Controller not initialized"
             )
-        await controller.register.remote(body)
+        try:
+            await controller.register.remote(body)
+        except Exception as e:
+            raise HTTPException(status_code=500, detail="Cannot register model, please contant the administrator")
```

Comment:
typo here, "contant" should be "contact"?

Reply:
Fixed